### PR TITLE
Change map colors

### PR DESCRIPTION
**Pull request by sharond106**
Website: https://sharondong-step-2020.appspot.com/pics.html

I just changed the colors of the default map and had to store the colors in another javascript file.

New map color: https://screenshot.googleplex.com/h3JCLRdGLg8

### CODE REVIEW

File: portfolio/src/main/webapp/map_style.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    "featureType": "poi",
+    "elementType": "geometry",
+    "stylers": [
+      {
+        "color": "#eeeeee"
+      }
+    ]
+  },
+  {
+    "featureType": "poi",
```
[the remark this answers is not in the document]
Comment (sharond106) [reply portfolio/src/main/webapp/map_style.js:53]: This code was generated for me from a page on google map's documentation. I don't think I can change the names. I'll add a comment and a link to it

File: portfolio/src/main/webapp/map_style.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
+var style = [
+  {
+    "elementType": "geometry",
```
[the remark this answers is not in the document]
Comment (sharond106) [reply portfolio/src/main/webapp/map_style.js:3]: Is adding the link to the exact documentation good or should I also copy comments from their directly into my file? The link is here: https://developers.google.com/maps/documentation/javascript/style-reference

[the remark this answers is not in the document]
Comment (vkkpng-g) [reply portfolio/src/main/webapp/map_style.js:5]: I think adding a link is good enough

File: portfolio/src/main/webapp/map_style.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (vkkpng-g) [reply portfolio/src/main/webapp/map_style.js:55]: no worries, reading the document was good enough for context what poi is

**Pull request merged**